todo_app/lib/models/todo.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:uuid/uuid.dart';
+
+const uuid = Uuid();
+
+enum Priority { urgent, normal, low }
```

Comment:
Az enum-okra is érvényes, hogy mindig külön file-okban tároljuk őket. Ez növeli az átláthatóságot, valamint ha nem csak A class hanem  B class is használja az enumot akkor a fordító nem fogja feleslegesen a B class mellé fordítani az A classot is, hanem csak az enumot.